src/test/java/uk/nhs/digital/nhsconnect/nhais/model/edifact/PersonNameTest.java

```diff
@@ -47,7 +47,7 @@ public void When_BuildingNameWithTypeOnly_Then_ReturnCorrectValue() {
 
     @Test
     public void When_BuildingEmptyName_Then_ReturnEmptySegment() {
-        var expectedValue = "PNA+PAT+'";
+        var expectedValue = "PNA+PAT'";
```

Comment:
We have code now to remove empty trailing fields

Reply:
I don't think empty segment like this would be valid